src/petname.c

```diff
@@ -0,0 +1,130 @@
+//
+// Copyright(C) 2019 Jonathan Dowland
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//     Generate a randomized, private, memorable name for a Player
+//
+
+#include <stdlib.h>
+#include <time.h>
+#include <string.h>
+
+static char *adjectives[] = {
```

Comment:
`static const char *const adjectives[]` to save @turol some work. :grin:

Reply:
Also, does it probably make sense to sort them alphabetically?

Reply:
Good idea re `const`. I did a quick bit of research and we can use two `const`s here, because the outer array and the inner strings are meant to be immutable. I think this might be true in loads of other places so, there might be another opportunity for @turol to touch every file again ;)

I didn't bother sorting the arrays. I can see the reason for it: to avoid duplicates. But, it would mean the Spice Girls would be separated from each other. And it's useful to group the nouns by game.